jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ZipkinSender.java

```diff
@@ -125,11 +124,6 @@ public int append(Span span) throws SenderException {
     return n;
```

Comment:
strictly speaking `this.queuedSizeInBytes` member var. isn't necessary as it's being recalculated every time per L97. In our UDPSender it was necessary because we did not keep re-calculating the full size, it was always `+= spanSizeInBytes`